e2e/react/src/react-package.test.ts

```diff
@@ -178,6 +160,37 @@ describe('Build React libraries and apps', () => {
       expect(parentLibOutput).toContain(`Bundle complete: ${childLib2}`);
     });
 
+    it('should support --format option', () => {
+      const childLibOutput = runCLI(`build ${childLib} --format cjs,esm,umd`);
+
+      updateFile(
```

Comment:
is the order correct here? first we build the lib, and then we change it?

Reply:
Er, yeah you're right!